File: app/src/main/java/ai/elimu/analytics/receiver/BootReceiver.java
package ai.elimu.analytics.receiver;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

import ai.elimu.analytics.task.TaskInitializer;

public class BootReceiver extends BroadcastReceiver {

    @Override
    public void onReceive(Context context, Intent intent) {
        Timber.i("onReceive");

        TaskInitializer.initializePeriodicWork(context);
    }
}

File: app/src/main/java/ai/elimu/analytics/receiver/BootReceiver.kt
package ai.elimu.analytics.receiver

import ai.elimu.analytics.task.TaskInitializer
import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import timber.log.Timber

class BootReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        Timber.i("onReceive")

        TaskInitializer.initializePeriodicWork(context)
    }
}
